module/programs/helper.js

```diff
@@ -1019,6 +1020,120 @@ module.exports = class ProgramsHelper {
     });
   }
 
+  /**
+   * Program details or cound o programs based on user.
+   * @method
+   * @name userProgram
+   * @param {String} reqesteduserId - userId .
+   * @param {String} stats          - condition to check stats or details .
+   * @returns {Object}              - Details of the program or count of the program.
+   */
+
+  static userProgram(reqesteduserId, stats = true) {
```

Comment:
why stats is true ? @praveenKDass

Reply:
Added as default value..if we pass arg as false it will take or else it will not but removed now @ankit1shahu